### DISCUSSION TITLE
How to add another pretrained model?

### DISCUSSION THREAD

**Discussion by huuduongnguyen** [PaddlePaddle/PaddleOCR#13101] [Q&A]
I have a pretrained model out there for handwriting detection and recognition, because this model seems cannot do that. Please can you help me to show me how to add it. Or guide me to train the model cause I have dataset too (Vietnamese language). Thank you so much!

**Comment by SWHL** [comment]
Text Detect: https://github.com/PaddlePaddle/PaddleOCR/blob/main/doc/doc_en/detection_en.md
Text Rec: https://github.com/PaddlePaddle/PaddleOCR/blob/main/doc/doc_en/recognition_en.md

**Reply by huuduongnguyen** [reply]
Thanks a lot, also is there any way that can help me output the word in UTF-8 encoding? Because it can read Vietnamese well, but the output seems glitch by encoding the word
